docs/administrator-guide/notification-management.md

```diff
@@ -54,6 +54,20 @@ If you’d like to see support for additional events, [let us know](https://matt
 
 ![This is the Channel Subscription modal](../.gitbook/assets/channel-subscriptions-modal.png)
 
+## How to create and use the subscription modal?
+
+1. Type the `/jira subscribe` command to create the subscription in the particular channel.
```

Comment:
Please add a recorded video of how this modal works in the PR description, so that I can verify the written steps with the video.

Reply:
[JiraSubscriptionModalWorking.webm](https://user-images.githubusercontent.com/107465508/213691139-75b3fe13-d4f6-43de-a85d-280a35e74328.webm)

Reply:
Please add this video in the PR description.

Reply:
```suggestion
1. Type the `/jira subscribe` command to open the "Create subscription" modal in the particular channel.
```